docs-src/concepts/what-is-a-logger.md

```diff
@@ -0,0 +1,33 @@
+---
+id: what-is-a-logger
+title: What is a Logger?
+sidebar_label: Logger
+description: A Logger provides an API containing the logic for recording runtime behavior.
+tags:
+  - term
+  - explanation
+  - logger
+  - api
+  - sdks
+---
+
+A logger provides an API containing the logic needed to record details about the application behavior at runtime.
+The output posts timestamps alongside the logged values, making it easier to evaluate the state and flow of [Workflows](/concepts/what-is-a-workflow) and [Activities](/concepts/what-is-an-activity).
+
+[Temporal SDKs](/concepts/what-is-a-temporal-sdk) provide both a logging API and a built-in logger implementation.
+The logging implementation provided by the SDK emits messages to the standard output of the [Worker](/concepts/what-is-a-worker) executing the code.
```

Comment:
This is not true of all SDKs. Many just use existing APIs and default to no logs.

Reply:
@cretz Is there a place where the details are documented, at least internally? 

I'm most familiar with the Go SDK, which works as described. The Java SDK provides a [replay-aware logger](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/internal/logging/ReplayAwareLogger.html), which the API docs suggest is accessible via `workflow.getLogger`, although there is no Activity logger. My understanding is that the TypeScript SDK recently added a logger. I don't know about the others. 

Perhaps since there is so much variation in the logging support offered by each SDK, documenting how it works might be better done as part of each SDK-specific dev guide.

Reply:
> Is there a place where the details are documented, at least internally?
Each SDK is different here. I am not sure we document them outside of SDK reference (even if we do there). I'd totally support documenting the SDK-specific details. That's unrelated to a general page.

> I'm most familiar with the Go SDK, which works as described. The Java SDK provides a [replay-aware logger](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/internal/logging/ReplayAwareLogger.html), which the API docs suggest is accessible via workflow.getLogger, although there is no Activity logger. My understanding is that the TypeScript SDK recently added a logger. I don't know about the others.

This comment is very specifically on the section stating that the implementation provided by the SDK emits to standard output. Not only is the standard output part not necessarily true. Python and .NET and Java all use ecosystem native logging that may not log anything by default (e.g. slf4j I think does nothing unless you include a JAR that has an impl).

> Perhaps since there is so much variation in the logging support offered by each SDK, documenting what is available and how to use it might be better done as part of each SDK-specific dev guide.

Exactly! I made this comment just below at https://github.com/temporalio/documentation/pull/2255#discussion_r1280933635